python/lsst/ts/m2/csc.py

```diff
@@ -771,7 +792,7 @@ def lookUpForces(self):
         myfa = np.zeros(self.n_actuators)
         myff = np.zeros(self.n_actuators)
 
-        lut_angle = -270.0 + self.zenith_angle
+        lut_angle = 90 - self.zenith_angle
```

Comment:
Another peaky detail, make sure `90` is a float.

```
lut_angle = 90. - self.zenith_angle
```

It avoids the trouble of defining an `int` than converting it to a `float`...